### PR TITLE
Fix zone feedback and killcount

### PR DESCRIPTION
**Pull request by manucamejo**
- Fix kill count
- Fix zone feedback

### CODE REVIEW

File: lib/dark_worlds_server/runner_supervisor/runner.ex [hunk 1]
```diff
@@ -482,4 +482,11 @@ defmodule DarkWorldsServer.RunnerSupervisor.Runner do
          {:loot, killed_id} | tail
        ]),
        do: [%{killed_by: 1111, killed: killed_id} | transform_killfeed_to_game_killfeed(tail)]
+
+  def transform_effects_to_game_effects([]), do: []
+
+  def transform_effects_to_game_effects([{%{name: "damage_outside_area"}, :zone} | tail]),
+    do: [{6, 0} | transform_effects_to_game_effects(tail)]
```
Comment (AminArria) [inline lib/dark_worlds_server/runner_supervisor/runner.ex:489]: Love the magic number value 😄

File: native/game_backend/src/game.rs [hunk 2]
```diff
@@ -454,6 +455,19 @@ fn update_player_actions(players: &mut HashMap<u64, Player>, elapsed_time_ms: u6
...
 }
 
+fn update_kill_counts(players: &mut HashMap<u64, Player>, killfeed: &Vec<KillEvent>) {
+    killfeed
+        .iter()
+        .for_each(|kill_event| match kill_event.kill_by {
+            EntityOwner::Player(player_id) => {
+                if let Some(player) = players.get_mut(&player_id) {
+                    player.add_kill();
+                }
```
Comment (AminArria) [inline native/game_backend/src/game.rs:465]: Should we also have a `player.add_death/0` ?

File: lib/dark_worlds_server/runner_supervisor/runner.ex (hunk 1 above)
Comment (manucamejo) [reply lib/dark_worlds_server/runner_supervisor/runner.ex:489]: This should be fixed after we implement the new state, and remove the transform functions

File: native/game_backend/src/game.rs (hunk 2 above)
Comment (manucamejo) [reply native/game_backend/src/game.rs:465]: No, I think we are not going to show the death count, and also players can't re-spawn

**Review by mFragaBA — APPROVED** [review]
LGTM!

**Review by tvillegas98 — APPROVED** [review]
LGTM!

**Pull request merged**